src/main/java/com/artipie/Pie.java

```diff
@@ -62,7 +63,8 @@
     private static final Map<String, Function<Storage, Slice>> ADAPTERS = new MapOf<>(
         new MapEntry<>("file", FilesSlice::new),
         new MapEntry<>("npm", sto -> new NpmSlice(new Npm(sto), sto)),
-        new MapEntry<>("gem", GemSlice::new)
+        new MapEntry<>("gem", GemSlice::new),
+        new MapEntry<>("php", storage -> new PhpComposer("", storage))
```

Comment:
@olegmoz what is the first `""` argument here? Maybe we need to move it to yaml config?

Reply:
@g4s8 that's base path, figured that maybe all repo types will need to support it later. Anyway, added reading base path from config, required to change the way Slices are created. Please review